### PR TITLE
About the language problem of README.md 

### PR DESCRIPTION
**Pull request by yunteng9345**
Go language is becoming more and more popular in China. ent is a very powerful framework, and many Chinese people will learn various Go language frameworks。

### CODE REVIEW

**Comment by a8m** [comment]
Thanks for your contribution @yunteng9345, it's appreciated. 

@hantmac - any chance you can help reviewing this? 🙏

File: README_zh.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- **静态类型和显式API** - 使用代码生成静态类型和显式API，查询数据更加便捷。
+- **多存储驱动程序** - 支持MySQL, PostgreSQL, SQLite 和 Gremlin。
+- **可扩展** - 简单地扩展和使用Go模板自定义。
+
+## 快速安装
+```console
+go get entgo.io/ent/cmd/ent
+```
+
+使用[Go modules]进行正确安装，请访问[entgo.io website][entgo instal]。
```
Comment (roger2hk) [inline README_zh.md:23]:
```suggestion
请访问[entgo.io website][entgo instal]以使用[Go modules]进行正确安装。
```

File: README_zh.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## 文档和支持
+开发和使用ent的文档请参照： https://entgo.io
+
+讨论问题和支持, [创建一个issue](https://github.com/facebook/ent/issues/new/choose) 或者加入我们的Gopher Slack(Slack软件,类似于论坛)[讨论组](https://gophers.slack.com/archives/C01FMSQDT53)
```
Comment (roger2hk) [inline README_zh.md:28]:
```suggestion
如要讨论问题和支持, [创建一个issue](https://github.com/facebook/ent/issues/new/choose) 或者加入我们的Gopher Slack(Slack软件,类似于论坛)[讨论组](https://gophers.slack.com/archives/C01FMSQDT53)  
```

**Review by roger2hk — CHANGES_REQUESTED** [review]
Please rename `README_zh.md` to `README_zh_CN.md` for zh_TW, zh_HK, zh_MO and zh_SG locale support later.

**Comment by yunteng9345** [comment]
> Please rename `README_zh.md` to `README_zh_CN.md` for zh_TW, zh_HK, zh_MO and zh_SG locale support later.

好想法！

**Comment by a8m** [comment]
Thanks for the review @roger2hk, it's appreciated!

**Pull request merged**